src/integration/pages/articlePage/pidgin.test.js

```diff
@@ -1,5 +1,5 @@
 /**
- * @pathname /pidgin/tori-51745682
+ * @pathname /pidgin/articles/cwl08rd38l6o
```

Comment:
NB: the previous pathname `/pidgin/tori-51745682`  was an STY

Reply:
Thanks for fixing